docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst

```diff
@@ -272,41 +275,113 @@ Where,
+GBD reports RRs and PAFs for years of life lost (YLLs) and years lived with disability (YLDs).
+YLLs are the number of deaths (due to a cause) multiplied by the average remaining life expectancy
+(which is a constant for a given age and sex group).
+YLDs are the number of prevalent cases of a cause multiplied by the disability weight
+of that cause, which is a constant for a given cause.
+Therefore
 
-        But maybe this belongs in the PAF section?
+.. math::
+
+  RR_\text{prevalence} = RR_\text{YLDs}
+
+  PAF_\text{prevalence} = PAF_\text{YLDs}
+
+  RR_\text{CSMR} = RR_\text{YLLs}
+
+  PAF_\text{CSMR} = PAF_\text{YLLs}
+
+where CSMR is the cause-specific mortality rate for a given cause.
 
-    Relevant ticket in backlog: https://jira.ihme.washington.edu/browse/SSCI-1152 
+However, if we have a dynamic SIS model for a cause, we are interested in applying risk effects to
+the transitions in this diagram:
+
+.. graphviz::
+
+  digraph {
+    rankdir=LR
+    node [shape=box]
+    S [label="Susceptible"]
+    I [label="Infected"]
+    D [label="Dead"]
+    S -> I [label="Incidence rate"]
+    I -> S [label="Remission rate"]
+    I -> D [label="Excess mortality rate"]
+  }
+
+The RR and PAF we have on prevalence (aka the number of people in the infected state)
+might apply to either of the incidence and remission rates, or some combination.
+Intuitively, the fact that more people with the risk exposure have the disease might be because
+more of them *get* the disease, or because it takes longer for them to *recover* from it.
+
+.. note::
+
+  Technically, mortality also matters: if the risk exposure causes people to die more quickly
+  if they have the disease,
+  that will lower the effect of the risk exposure on prevalence (since people who die are no
+  longer prevalent cases).
+  However, this effect is small when the mortality rate overall is small.
+  See TODO below to make this precise.
+
+We typically apply the risk effect to the incidence rate, but this is a modeling choice.
+With this choice:
+
+.. math::
+
+  RR_\text{remission} = 1
+
+  PAF_\text{remission} = 0
+
+  RR_\text{incidence} = RR_\text{prevalence}
+
+  PAF_\text{incidence} = PAF_\text{prevalence}
+
+The RR and PAF we have on CSMR doesn't directly fit anywhere into our diagram, because CSMR is a mortality rate among
+the total population, and our mortality transition only applies to the infected population.
+Intutively, the CSMR measures *overlap* with the prevalence measures; there could be more deaths among
+people with the risk exposure because they are more likely to have the disease, even if
+they are no more likely to die from it once they have it.
+
+We've developed a simple equation to adjust the CSMR RR to apply to the excess mortality rate:
+
+.. math::
+  
+  RR_\text{excess mortality} = \frac{RR_\text{CSMR}}{RR_\text{incidence}}
```

Comment:
Add note about not having covariance estimated by GBD between the numerator and denominator